templates/case-study/managed-kafka-for-smart-safety.html

```diff
@@ -0,0 +1,203 @@
+{% extends "case-study/base_case-study.html" %}
+
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+
+{% from "macros/_macros-text-list.jinja" import text_list %}
+{% from "macros/_macros-highlights.jinja" import highlights %}
+{% from "macros/_macros-image.jinja" import image_wrapper %}
+{% from "macros/_macros-text.jinja" import text %}
+{% from "_macros/vf_quote-wrapper.jinja" import vf_quote_wrapper %}
+
+{% block title %}Managed Kafka for smart safety{% endblock %}
+
+{% block meta_description %}
+  Learn how this smart safety technology leader unlocked immense annual savings with Canonical’s Managed Kafka.
+{% endblock %}
+
+{% block meta_company_name %}Managed Kafka for smart safety{% endblock %}
```

Comment:
Where does this value come from?

Reply:
I copied the block tag from another case study and the value is in the cop doc for the breadcrumb, otherwise it says "Default company"

Reply:
I've just left a note for UX in the doc about it. Apart from that everything is good to go. I will message you when you can merge this